Dockerfile

```diff
@@ -17,7 +17,7 @@ FROM perl:5.40.0-slim-threaded-buster
 #
 
 ARG LATEXINDENT_VERSION
-ENV LATEXINDENT_VERSION ${LATEXINDENT_VERSION:-V3.24.7}
+ENV LATEXINDENT_VERSION=${LATEXINDENT_VERSION:-V3.24.7}
```

Comment:
I corrected the deprecated syntax since it causes the following warning while building:
```
 1 warning found (use docker --debug to expand):
 - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 20)
```